app/src/main/java/com/cornellappdev/scoop/ui/screens/PostScreen.kt

```diff
@@ -1,5 +1,6 @@
 package com.cornellappdev.scoop.ui.screens
```

Comment:
implement the changes I showed you during work session so that going back on the first screen leads back to the original home screen instead of a screen that just states trip Details. Will need to update the get a ride button to go to 0 instead of 1 now.